src/main/java/org/venice/piazza/servicecontroller/Application.java

```diff
@@ -113,6 +118,16 @@ public static void inspectSprintEnv(ApplicationContext ctx) {
 		}
 	}
 
+	@Bean(name = "UpdateJobsQueue")
```

Comment:
This is because we have two beans declared of the same Type (note the second Bean below). This helps us denote the two individually so that Spring knows which Bean to autowire in a given situation. 

The reason the other projects don't contain this is because most projects only send one type of message -- the Update.  Service controller is different in that it sends two types of messages, Update and Request. 

I'm open to better ways to handle this -- but this way seemed clean to me. I like the explicit naming such that there won't be any confusion as to which bean is being used.